src/server/index.js

```diff
@@ -2,9 +2,16 @@
 const express = require('express');
 require('dotenv').config();
 const app = express();
-
+const path = require('path');
 console.log(process.env.DB_URL);
 app.use(express.static('dist'));
+// app.get('/signup', (req, res) =>{
+//   res.sendFile('index.html', {root: path.join(__dirname, '../../dist')});
+//   res.send('hello from s');
+// });
+app.get('/signup', (req, res) =>{
```

Comment:
do you want to define an endpoint for each screen? We have better solutions for this. for example express accepts the '*' character as route and it will match everything.
http://expressjs.com/en/4x/api.html#router.all